modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1911,6 +1911,48 @@ definition {
 		}
 	}
 
+	@description = "LPS-187081 - Verify the user cannot delete the only custom field in an object definition"
+	@priority = 5
+	test CannotDeleteSoleCustomField {
+		
+		task ("Given a custom object published with a custom field") {
```

Comment:
`Given a custom object with a custom field`

I put the wrong text on the ticket, that's my bad